### ISSUE TITLE
FAILURE: Build Failed with an Exception

### ISSUE THREAD

**Issue by NickDeBeenSAE** [NationalSecurityAgency/ghidra#5696]
![image](https://github.com/NationalSecurityAgency/ghidra/assets/60248725/9aab0555-b4f6-4130-b1b4-3090ede00c23)
 Happens even if I have installed Gradle, OpenJDK, and even if I created a configuration file manually.

**Comment by NickDeBeenSAE** [comment]
It fails to run also, even if I have Gradle installed:

./ghidraRun
Cannot lauch from a repo because Ghidra has not been compiled with Eclipse or Gradle

**Comment by ryanmkurtz** [comment]
Delete your `ghidra.repos.config` file and run `gradle -I gradle/support/fetchDependencies.gradle init`.

If you exactly follow the build steps in the [README.md](https://github.com/NationalSecurityAgency/ghidra/blob/master/README.md) file, you should have no problems.

**Comment by NickDeBeenSAE** [comment]
Ok, I had to jump through a fair amount of hoops, but I eventually installed Ghidra just fine.

**Comment by ryanmkurtz** [comment]
I'm curious what the hoops were.  Were you in an offline environment?

**Comment by NickDeBeenSAE** [comment]
No, I was online.

I do need help identifying where known values are in a file.

I.e., I am trying to use Ghidra to find said known value.

In my case $21,766 or just 21766.

This value is stored in a PS3 save file.

The PS3 save file is PowerPC instruction based.

This is MIPS32 architecture.

But, I digress, so I'll get back on track.

Installing on Windows is different to Linux.

I had to download Gradle, and install it, I then downloaded Adoptium OpenJDK 20 and installed that.

I then set Gradle as an Environment variable as per Ghidras Installation instructions.

I then simply used ``gradle --init`` which downloaded and built Ghidras dependencies.

Lastly, for Linux, and this is what I meant when I said I had to jump through a fair few amount of hoops, I git cloned the repository, I cd into its directory (by the way, I just downloaded the latest release on Windows, then opened the directory in a terminal), I then had to manually download the repo of Gradle, Maven and OpenJDK 20 and 17 (for this to properly work with 20 for some reason) then used the command you mentioned.

I simply used ``gradle buildGhidra`` but it partially worked.

So I used ``gradle -prepDev``

Yup, we are away now, finally after 3 hours of headache.

Easy, perfect, I used the same command you replied to my initial issue with, and package built, program ready to go, and does the program run though?...... kind of.

It exited telling me to run it in the foreground.....

Ok, here we go, more headache, more enraging irritations of programming.....

So, I opened the Ghidra directory (inside the initially cloned ghidra directory), then cd into RuntimScripts, Linux, then I had to edit the god damn script so its launch parameter was set to run in foreground for it to work, then I had to re-build Ghidra using ``gradle buildGhidra`` once I returned to ghidra directory, this failed.

I then used ``gradle wrapper``.......................

It built the updated program for Ghidra to launch in the foreground............

Ok, the end right? No more pulling my hair out? No more pain?

Yeah, this is when your sent command 1 final time and then everything worked.

Ghidra ran for the first time on my system perfectly.
